DeepSpeech.py

```diff
@@ -670,12 +671,77 @@ def __call__(self, progress, data, **kwargs):
         log_info('FINISHED optimization in {}'.format(datetime.utcnow() - train_start_time))
     log_debug('Session closed.')
 
+def finetune_lm_params():
+    original_alpha, original_beta = FLAGS.lm_alpha, FLAGS.lm_beta
+    tmp_csv_path = os.path.join(FLAGS.finetune_temp_dir, '__TMP_FINETUNE_SAMPLE__.csv')
```

Comment:
Use the `tempfile` module instead of the temp dir flag and hardcoding a file name.